lua/lspconfig/server_configurations/eslint.lua

```diff
@@ -112,6 +112,11 @@ return {
         name = vim.fn.fnamemodify(new_root_dir, ':t'),
       }
 
+      -- Support flat config
+      if #vim.fs.find('eslint.config.js', { path = new_root_dir, type = 'file' }) == 1 then
```

Comment:
this not work with link type . and missing stop condition like `stop = vim.env.HOME`

Reply:
No, it only needs to find in the `new_root_dir` that’s resolved by `root_dir`, with the [default of `upward=false`](https://neovim.io/doc/user/lua.html#:~:text=upward%20(boolean%2C%20default%20false)) hence `stop` is unnecessary.

I'm not clear what the link type refers to, how should we fix it?

Reply:
system link type  `ln -s`